src/object-util/object-util.spec.ts

```diff
@@ -26,4 +26,41 @@ describe('ObjectUtil', () => {
       expect(result).toBeNull();
     });
   });
+  describe('isNullish', () => {
+    it('should return true', () => {
+      const value = null;
+      const result = ObjectUtil.isNullish(value);
+      expect(result).toBe(true);
+    });
+    it('should return true', () => {
+      const value = undefined;
+      const result = ObjectUtil.isNullish(value);
+      expect(result).toBe(true);
+    });
```

Comment:
같은 류의 테스트는 묶어서 실행해도 됩니다.
이정도 간단한 내용은 변수 할당 없이 inline으로 작성해도 되겠습니다
```suggestion
    it('should return true', () => {
      expect(ObjectUtil.isNullish(null)).toBe(true);
      expect(ObjectUtil.isNullish(undefined)).toBe(true);
    });
```